pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSchemas.java

```diff
@@ -34,6 +38,7 @@ public CmdSchemas(PulsarAdmin admin) {
         jcommander.addCommand("get", new GetSchema());
         jcommander.addCommand("delete", new DeleteSchema());
         jcommander.addCommand("upload", new UploadSchema());
+        jcommander.addCommand("pojo", new PojoSchema());
```

Comment:
`pojo` is not an "action" here. why can't we add the options to `upload`, which I think it is the better place.

Reply:
we can probably rename it extract if that makes more sense if we combine it with it upload the parameters combinations they will have to be manually managed since there we no longer have required parameter annotations as they will be two distinct groups of params.
@srkukarni